packages/core/src/Stores/client-store.js

```diff
@@ -721,6 +722,10 @@ export default class ClientStore extends BaseStore {
         return this.account_status?.status?.includes('financial_assessment_notification');
     }
 
+    get is_poa_older_than_six_months() {
+        return this.account_status?.status?.includes('poa_expired') || true;
```

Comment:
lets not push the `|| true` condition to github. let it work when the backend code is ready.

Reply:
Yes I agree with @shaheer-deriv.
@sanjam-deriv Did you confirm this status with BE team members?

Reply:
Hey @matin-deriv @shaheer-deriv for now the backend card is still in progress

Reply:
> lets not push the `|| true` condition to github. let it work when the backend code is ready.

If I remove this the build will not trigger as the api call does not exist

Reply:
Removed the condition with a todo on the flag